packages/constants/providers.ts

```diff
@@ -0,0 +1,80 @@
+import * as _ from 'lodash';
+import { SecretModel } from "../models";
+import { getAPIVersion } from "../shared/selectors/common";
+import { NS_NOOBAA_TYPE_MAP, NS_PROVIDERS_NOOBAA_MAP } from './noobaa';
+
+export enum BC_PROVIDERS {
+  AWS = 'AWS S3',
+  S3 = 'S3 Compatible',
+  PVC = 'PVC',
+  GCP = 'Google Cloud Storage',
+  AZURE = 'Azure Blob',
+  IBM = 'IBM COS',
+  FILESYSTEM = 'Filesystem',
+}
+
+export const AWS_REGIONS = [
+  'us-east-1',
+  'us-east-2',
+  'us-west-1',
+  'us-west-2',
+  'ca-central-1',
+  'eu-central-1',
+  'eu-west-1',
+  'eu-west-2',
+  'eu-west-3',
+  'eu-north-1',
+  'ap-east-1',
+  'ap-northeast-1',
+  'ap-northeast-2',
+  'ap-northeast-3',
+  'ap-southeast-1',
+  'ap-southeast-2',
+  'ap-south-1',
+  'me-south-1',
+  'sa-east-1',
+];
+
+export type nsSpecProvider = typeof NS_PROVIDERS_NOOBAA_MAP[keyof typeof NS_PROVIDERS_NOOBAA_MAP];
+export type nsSpecType = typeof NS_NOOBAA_TYPE_MAP[keyof typeof NS_NOOBAA_TYPE_MAP];
+
+export const secretPayloadCreator = (
+  provider: string,
+  namespace: string,
+  secretName: string,
+  field1: string,
+  field2 = '',
+) => {
+  const payload = {
+    apiVersion: getAPIVersion(SecretModel),
+    kind: SecretModel.kind,
+    stringData: {},
+    metadata: {
+      name: secretName,
+      namespace,
+    },
+    type: 'Opaque',
+  };
+
+  switch (provider) {
+    case BC_PROVIDERS.AZURE:
+      payload.stringData = {
+        AccountName: field1,
+        AccountKey: field2,
+      };
+      break;
+    case BC_PROVIDERS.IBM:
+      payload.stringData = {
+        IBM_COS_ACCESS_KEY_ID: field1,
+        IBM_COS_SECRET_ACCESS_KEY: field2,
+      };
+      break;
+    default:
+      payload.stringData = {
+        AWS_ACCESS_KEY_ID: field1,
+        AWS_SECRET_ACCESS_KEY: field2,
+      };
+      break;
+  }
+  return payload;
+};
```

Comment:
this will go under utils not constants

Reply:
Done, moved to utils

Reply:
as you can see one is inside shared and the other is a standalone util directory. 

![image](https://user-images.githubusercontent.com/22158580/164230557-d8415163-3139-4f90-b30e-f947bfdc3ce7.png)


my question is do we need to keep both? if yes then where do we need to put what?